### PR TITLE
add subscription class

### PR DESCRIPTION
**Pull request by ephraimbuddy**
This will enable subscription from paystackapi

### CODE REVIEW

File: paystackapi/subscription.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        """
+        Fetch subscription.
+
+        Args:
+            subscription_id: subscription id.
+
+        Returns:
+            Json data from paystack API.
+        """
+        return cls().requests.get('subscription/{subscription_id)'.format(**locals()))
```
Comment (andela-sjames) [inline paystackapi/subscription.py]: This is really cool. Please, can we have test up for this resource, this looks good to me though.

Comment (webcalc) [reply paystackapi/subscription.py]: Sure...

Comment (webcalc) [reply paystackapi/subscription.py]: Done

Comment (andela-sjames) [reply paystackapi/subscription.py]: I still can see any test for this resource or am I missing something

Comment (andela-sjames) [reply paystackapi/subscription.py]: I still can't see any test for this resource or am I missing something

**Comment by andela-sjames** [comment]
Also, can the PR be raised against the `develop` branch? Thanks.

**Comment by webcalc** [comment]
Yeah. I'll do that. Thanks

File: paystackapi/subscription.py (hunk 1 above)
Comment (webcalc) [reply paystackapi/subscription.py]: Sorry. Must have made a mistake. First time stuff :) .

**Pull request closed without merging**